src/connector/src/sink/remote.rs

```diff
@@ -661,13 +662,11 @@ impl SinkWriter for CoordinatedRemoteSinkWriter {
         if is_checkpoint {
             // TODO: add metrics to measure commit time
             let rsp = self.stream_handle.commit(epoch).await?;
-            rsp.metadata
-                .ok_or_else(|| {
-                    SinkError::Remote(anyhow!(
-                        "get none metadata in commit response for coordinated sink writer"
-                    ))
-                })
-                .map(Some)
+            Ok(Some(rsp.metadata.unwrap_or_else(|| SinkMetadata {
+                metadata: Some(sink_metadata::Metadata::Serialized(SerializedMetadata {
```

Comment:
Why do we change to not return any error here?

Reply:
This is because it is a normal phenomenon for this commit to return "none". The commit of Redshift sink does not return any value, but we have adopted Coordinator method, so it is necessary to mock one here.